src/components/PageHeader.tsx

```diff
@@ -38,10 +42,12 @@ const StyledTruncatableHeading = styled.div<{ canBeTruncated: boolean }>`
 export interface PageHeaderProps extends TypographyProps {
   backPath?: string;
   children: string;
+  details?: { publicationType: string; publicationYear: string };
```

Comment:
Hvis vi noen gang trenger lignende heading andre plasser burde det være `details?: ReactNode`, men så lenge det _kun_ skal være på landing page går det for det samme

Reply:
ja, det er sant, men vi får endre det når vi trenger det (ref [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)) 👍